src/main/java/ua/yakaboo/pages/BasePage.java

```diff
@@ -0,0 +1,11 @@
+package ua.yakaboo.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
```

Comment:
Please, put WebDriver and Waits classes to BasePage and than you can use it in all your pages

Reply:
Done